pkg/addon/manager/manifests/templates/namespace.yaml

```diff
@@ -2,3 +2,7 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ .AddonInstallNamespace }}
+{{ if eq .AddonInstallNamespace "open-cluster-management-agent-addon" }}
```

Comment:
is this eq or ne?

Reply:
eq, if it is open-cluster-management-agent-addon, mark it as "deletion-orphan"